scripts/build-manylinux-wheels.sh

```diff
@@ -25,12 +25,25 @@ mkdir -p /io/temp-wheels
 find /io/temp-wheels/ -type f -delete
 
 # Iterate through available pythons.
-for PYBIN in /opt/python/cp3[6789]*/bin; do
-    "${PYBIN}/pip" install -q -U setuptools wheel nose --cache-dir /io/pip-cache
-    # Run the following in root of this repo.
-    (cd /io/ && USE_OMP=$USE_OMP "${PYBIN}/pip" install -q .)
-    (cd /io/ && USE_OMP=$USE_OMP "${PYBIN}/python" setup.py nosetests)
-    (cd /io/ && USE_OMP=$USE_OMP "${PYBIN}/python" setup.py -q bdist_wheel -d /io/temp-wheels)
+for PY in cp3{6,7,8,9,10}; do
+    if [[ $PLAT == manylinux2010* ]]; then
```

Comment:
Doesn't this require `=~` instead of `==`?

Reply:
No, it's a pattern match, not a regex.

Reply:
:man_facepalming: Thanks